audit/format.go

```diff
@@ -362,11 +363,12 @@ type AuditResponse struct {
 }
 
 type AuditAuth struct {
-	ClientToken string            `json:"client_token"`
-	Accessor    string            `json:"accessor"`
-	DisplayName string            `json:"display_name"`
-	Policies    []string          `json:"policies"`
-	Metadata    map[string]string `json:"metadata"`
+	ClientToken  string            `json:"client_token"`
+	Accessor     string            `json:"accessor"`
+	DisplayName  string            `json:"display_name"`
+	Policies     []string          `json:"policies"`
+	Metadata     map[string]string `json:"metadata"`
+	TokenNumUses int               `json:"token_num_uses,omitempty"`
```

Comment:
Agreed. I had reverted the name in `logical.Auth` struct. It makes sense to do it here as well.